daily-challenges/2024-09/14/hunghoang3110.spec.ts

```
@@ -0,0 +1,41 @@
+// Javascript
+function caculateBMI(weight, height) {
+    var bmi, phanloai;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)